package.json

```diff
@@ -49,5 +49,9 @@
 		"react-dom": ">=16",
 		"react-syntax-highlighter": "^15.5.0",
 		"uuid": "^9.0.0"
+	},
+	"dependencies": {
+		"moment": "^2.29.4",
```

Comment:
moment should be a peer dependency. postcss is already listed there. As a general rule, packages should never have direct dependencies, only dev and peer dependencies. Direct dependencies will cause rollup to try and package them into the package; we do not want that.